lib/omnibus/metadata.rb

```diff
@@ -63,6 +63,7 @@ def generate(path, project)
           iteration:        project.build_iteration,
           license:          project.license,
           version_manifest: project.built_manifest.to_hash,
+          license_content:  File.read(project.license_file_path),
```

Comment:
We probably want to project this with `File.exist?()` incase the project does not have this file present.